modules/virt-creating-custom-monitoring-label-for-vms.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-creating-custom-monitoring-label-for-vms_{context}"]
+= Creating a custom monitoring label for virtual machines
+
+To enable queries to multiple virtual machines from a single service, create a custom label and assign it to the virtual machine's YAML file. 
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+* Access to the web console for stopping and restarting a virtual machine.
+
+.Procedure
+* Edit the `template` spec of your virtual machine configuration file. The following example shows the label name `metrics` being added to the label `monitor`.
++
+[source,yaml]
+----
+spec:
+  template:
+    metadata:
+      labels:
+        monitor: metrics
+----
+
+* Stop and restart the virtual machine to create a new pod with the label name given to the `monitor` label.
+
+.Verification
+
+To verify that the `node-exporter` service is working and view the metrics for a virtual machine:
+
+. Expose the `node-exporter` service.
++
+[source,terminal]
+----
+$ oc expose service -n <node_exporter_service_name>
```

Comment:
Please change from `$ oc expose service -n <node_exporter_service_name>`  to `$ oc expose service -n <namespace> <node_exporter_service_name>`

Reply:
Updated the content.